src/Squirrel/DeltaPackage.cs

```diff
@@ -331,5 +352,40 @@ bool bytesAreIdentical(byte[] oldData, byte[] newData)
 
             return true;
         }
-    }
+
+		/// <summary>
+		/// Compares two files if they are identical. Source: https://stackoverflow.com/a/1359947
+		/// </summary>
+		/// <param name="first">First file for the comparison.</param>
+		/// <param name="second">Second file for the comparison.</param>
+		/// <returns>True if files are identical otherwise false.</returns>
+		bool filesAreEqual(FileInfo first, FileInfo second)
```

Comment:
Not critical, but would not it be better to compute hash of each file and then compare it :)

Reply:
Just small note. If the hash will be computed from the whole file then I think that this implementation will be much faster because the execution ends immediately when file bytes are different.